__tests__/integration/previewSaveRDF.test.js

```diff
@@ -2,35 +2,19 @@
 
 import pupExpect from 'expect-puppeteer'
 import { testUserLogin } from './loginHelper'
+import { fillInRequredFieldsForBibframeInstance } from './previewRDFHelper'
 
 describe('Previewing the RDF', () => {
   beforeAll(async () => {
     return await testUserLogin()
   })
 
+  beforeEach(async () => {
+    return await fillInRequredFieldsForBibframeInstance()
+  })
+
   it('builds the rdf and has dialog for saving', async () => {
-    expect.assertions(18)
-    await pupExpect(page).toClick('a', { text: 'BIBFRAME Instance' })
-    await pupExpect(page).toMatch('BIBFRAME Instance')
-
-    // Click on one of the property type rows to expand a nested resource
-    await pupExpect(page).toClick('a[data-id=\'title\']')
-    await pupExpect(page).toMatchElement('h5', { text: 'Work Title' })
-
-    // Fill in required element
-    await pupExpect(page).toClick('a[data-id=\'mainTitle\']')
-    await page.type('[placeholder=\'Preferred Title for Work (RDA 6.2.2, RDA 6.14.2) (BIBFRAME: Main title)\']', 'Hello')
-    await page.keyboard.press('Enter')
-
-    // Fill in required element
-    await page.type('[placeholder=\'Agent Contribution\']', 'Typing')
-    // wait until autosuggest has returned something to click on
-    await page.waitForSelector('#rbt-menu-item-0')
-    await pupExpect(page).toClick('#rbt-menu-item-0')
-
-    // Fill in required element
-    await page.type('[placeholder=\'Statement of Responsibility Relating to Title Proper (RDA 2.4.2)\'', 'World')
-    await page.keyboard.press('Enter')
+    expect.assertions(13) // An additional assertion is done in previewRDFSetup
```

Comment:
Change comment to read `fillInRequredFieldsForBibframeInstance`

Reply:
Done.